Server/Content/Sources/App/Controllers/ArticleController .swift

```diff
@@ -0,0 +1,25 @@
+import Fluent
+import Vapor
+
+struct ArticleController {
+    func all(req: Request) throws -> EventLoopFuture<[Article]> {
+        return Article.query(on: req.db).all()
+    }
+    
+    func getByID(req: Request) throws -> EventLoopFuture<[Article]> {
+        return Article.find(req.parameters.get("articleID"), on: req.db)
```

Comment:
You treat parameter name as `articleID` here, but in fact you pass it as `textID` when declaring a route.

Reply:
Also your function name is `getByID` here but you refer to it as `getById` when declaring routes, so your app wouldn't even compile here.